cmd/manager/main.go

```diff
@@ -115,6 +116,7 @@ func main() {
 	flag.BoolVar(&operatorControllerVersion, "version", false, "Prints operator-controller version information")
 	flag.StringVar(&systemNamespace, "system-namespace", "", "Configures the namespace that gets used to deploy system resources.")
 	flag.StringVar(&globalPullSecret, "global-pull-secret", "", "The <namespace>/<name> of the global pull secret that is going to be used to pull bundle images.")
+	flag.StringVar(&registriesConfig, "registries-config", "/etc/containers/registries.conf", "The local file path of the registries configuration file")
```

Comment:
hey @dtfranz, just a question - could you explain how this file would be initially created and then tracked?

In case of globalPullSecret, we set up a reconciler that is able to watch on the secret changes and then record changes in data in local authFile, but I don't see which part of the code would be responsible for working with the registries file.

also, NIT: maybe `registriesConfigPath` similar to how there's `certFile` and `keyFile`?

Reply:
The entire `/etc/containers` directory can be mounted into the pod. `/etc/containers` is the default configuration directory for `containers/image`

Given my comment about the likely need to read/hash the entire directory content, I would suggest dropping this flag and hardcoding the assumption of `/etc/containers` (Maybe there's a constant in the `containers/image` library we could use?)

Reply:
this may also be naive, but if it _can_ (optionally) be mounted then should we always assume it has been mounted in `Unpack`? Maybe it would be better to check if the file/directory exists first?